wp-content/plugins/core/src/Blocks/Deny_Blocks.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Project\Blocks;
+
+/**
+ * Class Deny_Blocks
+ *
+ * Provides an array of block names to remove from the editor.
+ */
+class Deny_Blocks {
```

Comment:
Should name this as an object e.g. noun/noun phrase. Ask yourself, "Can you hand me the <class name>?" and does it make grammatical sense? 

Can you hand me the Deny Blocks?

How about `Block_Denier`?

Reply:
`Denied_Blocks` maybe? That keeps with the previous convention of `Allowed_Blocks` and could help guide other developers that have been using the `Allowed_Blocks` functionality that is being removed with this PR.

Reply:
Allowed Blocks was also an improper name. There shouldn't be any plural classes. 

An opinionated article on the subject:
https://medium.com/swlh/an-opinionated-guide-to-naming-your-code-aimed-at-new-developers-6f769d60e098

How about `Block_Deny_List::add( $types )`?

That should be clear to any developer new and old what's happening at a glance

Reply:
Yes, `Block_Deny_List` is perfect.  Thanks. updating it now.

Reply:
Sorry was in a rush for an appointment before, but I also wanted to explain why this is important and why it makes sense.

While PHP-DI just does this behind the scenes, if you run `$deny_list = new Block_Deny_List( [] );` what is actually happening? 

You're creating an object (an instance of a class). Naming as a singular noun makes a lot more sense because you end up with a single instance of a class, say using `Animal()` as an example:

````php
$tiger = new Animal( 'tiger );
$bear = new Animal( 'bear' );
$chicken = new Animal( 'chicken' );
```
Would be strange to have a single instance of a plural object:

````php
// If it's plural, is this...a bunch of animals or what? It's just more confusing since you're creating a single instance of an Animal for each animal.
$tigers = new Animals( 'tiger' );
$bears = new Animals( 'bear' );
$chickens = new Animals( 'chicken' );
```